packages/devextreme/js/__internal/grids/grid_core/focus/m_focus.ts

```diff
@@ -205,10 +208,13 @@ export class FocusController extends core.ViewController {
     if (!this.isAutoNavigateToFocusedRow()) {
       this.option('focusedRowIndex', -1);
     }
-    return this._navigateToRow(key);
+    return this._navigateToRow(key, false);
   }
 
-  public _navigateToRow(key, needFocusRow?) {
+  public _navigateToRow(
+    key: any,
```

Comment:
Minor
Afair, there was some `export type Key = any`
It will not change anything for TS, but let's still use it for better readability